### PR TITLE
start helm

### PR DESCRIPTION
**Pull request by osnatashush**
for your reviewing

### CODE REVIEW

File: charts/app/templates/configmap.yaml
Comment (shlomosfez) [inline charts/app/templates/configmap.yaml:1]: remove this file

File: charts/app/templates/deployment.yaml [hunk 1]
```diff
@@ -60,20 +60,49 @@ spec:
...
-          envFrom:
             - secretRef:
                 name: {{ $fullname }}-secrets-env
           {{- end }}
+          {{- end }} 
+
+
+
+
+          # envFrom:
```
Comment (shlomosfez) [inline charts/app/templates/deployment.yaml:87]: remove this commented out code

File: charts/app/values.yaml [hunk 2]
```diff
@@ -138,3 +138,29 @@ serviceMonitor:
   enabled: false
   interval: 30s
   metricsPort: 9111
+
+
+configmaps:
+  enabled: false
+  names:
+    - configmap1
```
Comment (shlomosfez) [inline charts/app/values.yaml:146]: remove this

File: charts/app/values.yaml [hunk 3]
```diff
@@ -138,3 +138,29 @@ serviceMonitor:
...
+    - configmap2
+    - configmap3
+
+
+
+# secrets:
```
Comment (shlomosfez) [inline charts/app/values.yaml:152]: remove this

File: output.txt [hunk 4]
```diff
@@ -0,0 +1,146 @@
+---
```
Comment (shlomosfez) [inline output.txt:1]: remove this

**Review by shlomosfez — CHANGES_REQUESTED** [review]

File: osnat-values.yaml
Comment (shlomodoktorabc) [inline osnat-values.yaml:1]: remove this file

**Comment by osnatashush** [comment]
done

**Pull request merged**